__test__/user.test.js

```diff
@@ -0,0 +1,35 @@
+import app from "../app.js";
+import supertest from "supertest";
+import mongoose from "mongoose";
+import { config } from "dotenv";
+
+config();
+
+describe("users", () => {
+  beforeAll(async () => {
+    await mongoose.disconnect();
+    await mongoose.connect(process.env.MONGO_URI);
+  });
+
+  afterAll(async () => {
+    await mongoose.disconnect();
+  });
+
+  it("it returns 200 status code", async () => {
+    const res = await supertest(app).get("/users");
+    expect(res.statusCode).toEqual(200);
+  });
+  describe("register", () => {
+    test("/signup", async () => {
+      const res = await supertest(app).post("/users/signup").send({
```

Comment:
If you are adding some records to DB, after tests you should wipe test data

Reply:
yes it is my bad, I will correct it now